compiler/parser/Makefile

```diff
@@ -36,20 +36,14 @@ include $(COMPILER_ROOT)/make/Makefile.compiler.subdirrules
 
 ifdef CHPL_DEVELOPER
 CLEAN_TARGS += \
-	*.tab.cpp \
-	*.yy.cpp \
-	*.tab.h \
-	*.output \
+               bison-chapel.output
 
 endif
 
-lex.yy.cpp: chapel.lex chapel.tab.h
-	flex -o$@ chapel.lex
-
-chapel.tab.h chapel.tab.cpp: chapel.ypp
-	bison --defines=chapel.tab.h -v -t $<
-	@if [ `grep "conflicts:" chapel.output | wc -l` -ne 0 ]; then echo "PROBLEM: chapel.y contains conflicts"; exit 1; fi;
-
+parser: FORCE
+	bison chapel.ypp
+	@if [ `grep "conflicts:" bison-chapel.output | wc -l` -ne 0 ]; then echo "PROBLEM: chapel.ypp contains conflicts"; exit 1; fi;
+	flex chapel.lex
```

Comment:
Oh cool!  I noticed the addition of the header name later on but hadn't made the connection.  That seems like a nice cleanup.